cinema/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS

```

Comment:
There's a syntax error in the import statement. Replace the forward slash (/) with a comma (,) to correctly separate the imported modules: `from rest_framework.permissions import BasePermission, SAFE_METHODS`.